src/components/ExpansionItem.vue

```diff
@@ -1,12 +1,14 @@
 <template>
-  <q-expansion-item :header-inset-level="1" expand-separator icon="receipt" :label="$t('pages')" default-closed>
+  <q-expansion-item :header-inset-level="1" expand-separator :icon=props.icon :label="$t(props.page)" default-closed>
```

Comment:
you don't need props.icon or props.page, in template you have direct access to icon / page